curriculum/challenges/english/01-responsive-web-design/basic-css/override-class-declarations-by-styling-id-attributes.english.md

```diff
@@ -7,7 +7,7 @@ videoUrl: 'https://scrimba.com/c/cRkpDhB'
 
 ## Description
 <section id='description'>
-We just proved that browsers read CSS from top to bottom. That means that, in the event of a conflict, the browser will use whichever CSS declaration came last.
+We just proved that browsers read CSS from top to bottom in order of their declaration. That means that, in the event of a conflict, the browser will use whichever CSS declaration came last. Notice that if we even had put <code>blue-text</code> before <code>pink-text</code> in our <code>h1</code> element's classes, it would still look at the declaration order and not the order of their use!
```

Comment:
Use inline code tags instead — so how it got split into multiple lines now.

Reply:
I used the same inline code tags that are used in the file. After my first commit, it got split into multiple lines but now everything seems normal. Can you also check?

Reply:
https://github.com/freeCodeCamp/freeCodeCamp/pull/19046/files?short_path=a9e2e48#diff-a9e2e48966f94570baac37a969757d77

Reply:
When I click Files Edited to edit the file, preview shows correctly:
https://github.com/freeCodeCamp/freeCodeCamp/blob/36a5043189870ddd5eddd69534ad423f5f63aef2/curriculum/challenges/english/01-responsive-web-design/basic-css/override-class-declarations-by-styling-id-attributes.english.md

I also used the same inline code tags. What do you think is the problem here, can you help?

Reply:
<code> is not inline, backticks are.